Documentation/ApiOverview/RequestLifeCycle/RequestAttributes/Language.rst

```diff
@@ -20,7 +20,5 @@ Example:
     $locale = $language->getLocale();
 
 
-API
-===
-
-..  include:: /CodeSnippets/Manual/Entity/SiteLanguage.rst.txt
+..  seealso::
```

Comment:
Why link people to another page when it can alos be displayed here conveniently?

Reply:
The phpdomain directives, just like the confval are generatic link targets, in this case duplicate ones. If you define the same class twice, linking it with :php:class: cannot generate a unique target. We could however make something like a parameter for display-only mode so we can display the same class in two different locations. But that would be a change

Reply:
Sadly, this is then a regress regarding usability. One click to get to the API instead visible immediately (and using browser's back to return to this page). But if this is the way it must be, then it is.

Reply:
We can define this any way we want. We should however decide what is supposed to happen then. What happened before was: sphinx didn't give a warning and linked to the first class it found...

Reply:
I made an issue about this: https://github.com/TYPO3-Documentation/render-guides/issues/461